plugins/webp-uploads/helper.php

```diff
@@ -16,20 +16,26 @@
  * for example an image/jpeg can be converted into an image/webp.
  *
  * @since 1.0.0
+ * @since n.e.x.t Added support for AVIF.
  *
  * @return array<string, array<string>> An array of valid mime types, where the key is the mime type and the value is the extension type.
  */
 function webp_uploads_get_upload_image_mime_transforms() {
```

Comment:
Might as well since these lines are being touched:

```suggestion
function webp_uploads_get_upload_image_mime_transforms(): array {
```